lib/matplotlib/backends/backend_pgf.py

```diff
@@ -693,9 +693,9 @@ def draw_image(self, gc, x, y, im, transform=None):
                   interp, w, h, fname_img))
         _writeln(self.fh, r"\end{pgfscope}")
 
-    def draw_tex(self, gc, x, y, s, prop, angle, ismath="TeX", mtext=None):
+    def draw_tex(self, gc, x, y, s, prop, angle, *, mtext=None):
```

Comment:
Can we do this without deprecation?

Reply:
Unclear. It is not used and was deprecated from all other `draw_tex` methods in #17484. However, it seems to have been forgotten for the pgf backend.

Reply:
The deprecations were removed in #19900 and while it was touched, it does not seem like anyone noticed that the signature was different.

Reply:
Ok, let's try.